src/main/java/me/flashyreese/mods/sodiumextra/SodiumExtraMod.java

```diff
@@ -0,0 +1,9 @@
+package me.flashyreese.mods.sodiumextra;
+
+import net.minecraftforge.fml.common.Mod;
+
+@Mod(SodiumExtraMod.MOD_ID)
+public final class SodiumExtraMod {
+    public static final String MOD_ID = "sodiumextra";
+    public SodiumExtraMod() {}
+}
```

Comment:
Why

Reply:
Seriously? I did a whole write-up inthe [associated issue](https://github.com/dimadencep/sodium-extra-forge/issues/46) explaining it.